leaflet/index.d.ts

```diff
@@ -1418,34 +1418,46 @@ declare namespace L {
         className?: string;
     }
 
-    class InternalIcon extends Layer {
+    export interface IconOptions extends LayerOptions {
+        iconUrl: string;
+        iconRetinaUrl?: string;
+        iconSize?: PointExpression;
+        iconAnchor?: PointExpression;
+        popupAnchor?: PointExpression;
+        shadowUrl?: string;
+        shadowRetinaUrl?: string;
+        shadowSize?: PointExpression;
+        shadowAnchor?: PointExpression;
+        className?: string;
+    }
+
+    class Icon extends Layer {
+        static Default: IconDefaultSettings;
```

Comment:
Except for the `InternalIcon` class (that is an artifact from when I first wrote the definitions using interfaces), the old way is correct. `Default` is not an instance, but another class and that is the reason way it was defined inside a namespace. You can verify that in the source code and in the documentation. Please revert. You are welcome to remove the `InternalIcon` class, as it is not needed anymore and it doesn't exist anyways.